manual/manual_backend.rst

```diff
@@ -53,18 +53,18 @@ Knowing the requirements of each :py:class:`~pytket.backends.Backend` is handy i
 
 .. jupyter-execute::
 
-    from pytket import Circuit
+    from pytket import Circuit, OpType
     from pytket.extensions.qiskit import AerBackend
 
-    circ = Circuit(2, 2)
-    circ.Rx(0.3, 0).Ry(0.5, 1).CRz(-0.6, 1, 0).measure_all()
+    circ = Circuit(3, 2)
+    circ.H(0).Ry(0.25, 1)
+    circ.add_gate(OpType.CnRy, [0.74], [0, 1, 2]) # CnRy not in AerBackend gate set
+    circ.measure_all()
 
     backend = AerBackend()
-    if not backend.valid_circuit(circ):
-        compiled_circ = backend.get_compiled_circuit(circ)
-        assert backend.valid_circuit(compiled_circ)
+    compiled_circ = backend.get_compiled_circuit(circ) # Compile circuit to AerBackend
```

Comment:
```suggestion
    print("Circuit valid for AerBackend?", backend.valid_circuit(circ))
    compiled_circ = backend.get_compiled_circuit(circ) # Compile circuit to AerBackend
```
Maybe worth adding a validity check before compilation like this to make it clear that compilation did something.